### PR TITLE
Add controller prometheus metrics

### PR DESCRIPTION
**Pull request by mrogers950**

### CODE REVIEW

File: deploy/role.yaml [hunk 1]
```diff
@@ -34,6 +34,13 @@ rules:
   verbs:
   - get
   - create
+  - update
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - prometheusrules
+  verbs:
+  - create
```
Comment (JAORMX) [inline deploy/role.yaml:43]: why do we need these? I don't see in the source code where the operator would create a PrometheusRule

Comment (mrogers950) [reply deploy/role.yaml:43]: That's at https://github.com/openshift/file-integrity-operator/pull/184/files#diff-51bd220db414457ea4470ea5d71465bd07e1c789d156c0104e1fce2f3ddbcb1aR224

**Review by JAORMX — APPROVED** [review]
/lgtm

yeesss!!!

**Pull request merged**